jinahub/indexers/query/keyvalue/PostgreSQLQueryIndexer/__init__.py

```diff
@@ -0,0 +1,105 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Tuple, Generator, Dict
+
+import numpy as np
+from jina import Executor, requests, DocumentArray
+from jina.logging.logger import JinaLogger
+
+from jinahub.indexers.dbms.PostgreSQLIndexer import PostgreSQLDBMSHandler
+
+
+class PostgreSQLQueryIndexer(Executor):
```

Comment:
When I look at this, I wonder if we should have one PostgreSQLIndexer for indexing and query and add a `read_only` as `__init__` parameter. :D

Reply:
We can explore this in another PR. This one is growing quite a lot, since the integration test is touching a lot of parts.